.gitignore

```diff
@@ -144,7 +144,8 @@ src/*/_version.py
 .idea/
 .vscode/
 
-input/*
 !input/*/.gitkeep
-output/*
+!input/templates/RRED_Report_Template_Single_Category.docx
```

Comment:
ooh maybe this file is better in`tests/resources/input...`?

Reply:
Yep agreed, will move that.